<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>데이터조회</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>로그인</string>
    </property>
   </widget>
   <widget class="QListView" name="stockItemListView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>151</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>종목리스트</string>
    </property>
   </widget>
   <widget class="QPushButton" name="requestStockInfoButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>종목기본정보</string>
    </property>
   </widget>
   <widget class="QListView" name="trDataListView">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>911</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="requestBalanceButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>100</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>잔고조회</string>
    </property>
   </widget>
   <widget class="QPushButton" name="requestChartButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>일봉데이터</string>
    </property>
   </widget>
   <widget class="QComboBox" name="accountComboBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>계좌번호</string>
    </property>
   </widget>
   <widget class="QPushButton" name="requestContinuousButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>일봉연속조회</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1115</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
